src/style.css

```
@@ -0,0 +1,58 @@
+html {
+  margin: 0;
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  margin-top: 0;
+  margin-bottom: 0;
+  padding: 0;
+}
+
+.stars__star {
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  border-radius: 0.5px;
+  margin-right: 4px;
+  list-style: none;
+  display: flex;
+}
+
+.stars__star:not(:first-child) {
+  background-image: url(./images/star.svg);
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star {
+  background-image: url(./images/star.svg);
+}
```

```suggestion
.stars__star:not(:last-child) {
  margin-right: 4px;
}
```

Just move `background-image` to `stars__star` 👍